cmd/vts/main.go

```diff
@@ -11,13 +11,15 @@ import (
 	"github.com/veraison/common"
 	"github.com/veraison/trustedservices"
 	"google.golang.org/grpc"
+
+	_ "github.com/mattn/go-sqlite3"
 )
 
 // TODO this a very minimal "frontend" implementation.
 func main() {
 	configPaths := common.NewConfigPaths()
 
-	flag.Var(configPaths, "config", "Path to direcotory containing the config file(s).")
+	flag.Var(configPaths, "config", "Path to directory containing the config file(s).")
```

Comment:
Just a note, as doing formal thorough review. What does a //TO DO here reflect. Can we translate into a clear issue for a proper front end, if this is the ask here?

Reply:
I think we can probably remove the `TODO` and leave the rest of the comment.  @setrofim ?

Reply:
Yeah,  agreed -- probably just remove the `TODO` bit -- it's entended to reflect that the whole implementation is intended to be quick-and-dirty, rather than complete.